src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,23 @@
 package core.basesyntax;
 
+import  java.util.Random;
+
 public class ColorSupplier {
+    enum Colors {
```

Comment:
```suggestion
    enum Color {
```

and move this enum to the separate file